sonar-plugin-api/src/main/java/org/sonar/api/test/TestCase.java

```diff
@@ -44,16 +48,23 @@ public static Status of(@Nullable String s) {
   /**
    * Duration in milliseconds
    */
+  @CheckForNull
   Long durationInMs();
 
+  /**
+   * @deprecated since 5.2 not used
+   */
+  @Deprecated
   String type();
```

Comment:
:new: ![INFO](https://raw.githubusercontent.com/henryju/image-hosting/master/info.png)Do not forget to remove this deprecated code someday.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)